public/data/Mustafiz04.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Mustafiz Kaifee",
+  "type": "personal",
+  "bio": "Hi there!, I am backend developer at freecharge.",
+  "avatar": "https://avatars.githubusercontent.com/u/49086821",
+  "links": [
+    {
+      "name": "Check out my personal website",
+      "url": "https://mustafiz04.github.io/",
```

Comment:
Personal website link gives a **`Page not found 404 Error`**

Reply:
This is because the site is not coded in HTML, its complete JS and your supposed to run it from the terminal I believe, looking from the repo: https://github.com/Mustafiz04/Mustafiz04.github.io